packages/js/src/helpers/disableMetaboxTabs.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Disables a metabox tab.
+ *
+ * @param {string} tabId The id of the tab to disable.
+ *
+ * @returns {void}
+ */
+function disable( tabId ) {
+	const tab = document.querySelector( `#${ tabId }` );
+	if ( tab === null ) {
+		return;
+	}
+
+	tab.style.opacity = "0.5";
+	tab.style.pointerEvents = "none";
+	tab.setAttribute( "aria-disabled", "true" );
+
+	if ( tab.classList.contains( "yoast-active-tab" ) ) {
+		tab.classList.remove( "yoast-active-tab" );
+	}
+}
+
+/**
+ * Disables the metabox tabs.
+ *
+ * @returns {void}
+ */
+export function disableMetaboxTabs() {
+	const tabs = Array.from( document.querySelectorAll( '[id^="wpseo-meta-tab-"]' ) );
+
+	tabs.map( ( tab ) => {
+		disable( tab.id );
+	} );
```

Comment:
You create an array here, to then map over it, only to not use the results at all?
No testing is done on the following code snippet. But a NodeList does have some helpers like `forEach`.
https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

```suggestion
	document.querySelectorAll( '[id^="wpseo-meta-tab-"]' ).forEach( ( tab ) => {
		disable( tab.id );
	} );
```